functions/gooey_builder_workflow_tools.py

```
@@ -0,0 +1,429 @@
+                "background": BACKGROUND_PROP,
+            },
+        )
+
+    def call(
+        self,
+        title: str | None = None,
+        description: str | None = None,
+        run_url: str | None = None,
+        background: bool = False,
+    ) -> dict:
+        page_cls, sr, pr = self.get_current_runs(run_url)
```

Ruff (0.15.15)

[warning] Unpacked variable `page_cls` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)